tests/testthat/test-filter.R

```diff
@@ -0,0 +1,95 @@
+context("filter")
+
+test_that("Can run the filter", {
```

Comment:
Add a test vs the Kalman filter with the volatility model?

Reply:
Done in be13de8; this was actually pretty pleasant to write